app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @post = Post.where(author_id: @user.id).order(created_at: :desc)
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @comment = Comment.where(post_id: @post.id).order(created_at: :desc)
+  end
```

Comment:
- [ ] Kindly fix the error in show post details
![image](https://user-images.githubusercontent.com/24703171/179065196-2477fbef-9843-40ec-9b38-511e114c45d1.png)